clause-handlers.js

```diff
@@ -30,59 +28,102 @@ module.exports = {
 			}
 		}
 	},
-	columnParam: function columnParam(joinedBy, opts, column, comparison, param) {
+	columnParam(joinedBy, opts, value1, comparison, value2) {
```

Comment:
One change that I ran into that may or may not be cool:

before, I assumed that the first argument `column` was always just a static string to be inserted straight into the query text, and the second argument `param` was meant to be parameterized.

With this change, the first value is parameterized if it is a result object returned by a template tag, but not if it is a string.

Another way I could have done it was formalize that the first argument could only ever be a string/column name.

@saibotsivad 
@daytonlowell